tensorflow_addons/rnn/cell.py

```diff
@@ -366,3 +366,471 @@ def _create_norm_layer(self, name):
             gamma_initializer=self.norm_gamma_initializer,
             epsilon=self.norm_epsilon,
             name=name)
+
+
+@tf.keras.utils.register_keras_serializable(package='Addons')
+# class LayernormSimpleRNNCell(SimpleRNNCell, LayerNormalization):
+class LayernormSimpleRNNCell(keras.layers.SimpleRNNCell):
+    """Cell class for LayernormSimpleRNN.
+
+    Motivation:
+    - Drop-In Replacement for keras.layers.SimpleRNNCell
+    - demonstrate how to add keras.layers.LayerNormalization
+       to all RNNs by introducing the `use_layernorm` argument
+
+    References:
+    [1] Ba, Jimmy Lei, Jamie Ryan Kiros, and Geoffrey E. Hinton.
+        “Layer Normalization.” ArXiv:1607.06450 [Cs, Stat],
```

Comment:
Failed log on CI says you're using non-ascii char here. Change to
```
"Layer Normalization." ArXiv:1607.06450 [Cs, Stat],
```
would be okay.

https://source.cloud.google.com/results/invocations/c550e39b-e32a-4243-ad4a-97f65c158e6c/targets

Reply:
https://github.com/tensorflow/addons/pull/841/commits/c2d9971861faafe5aa451c45e178938afb3d26ea

Reply:
Thanks for the tip. changed to `"`